scripts/st2bootstrap-el7.sh

```diff
@@ -442,8 +504,12 @@ install_st2mistral() {
     sudo yum -y install ${PACKAGE_URL}
   fi
 
+  # Configure database settings
+  sudo crudini --set /etc/mistral/mistral.conf database connection "postgresql://mistral:${ST2_POSTGRESQL_PASSWORD}@localhost/mistral"
```

Comment:
Let's use `127.0.0.1` where possible instead of `localhost` as we agreed before.